package.json

```diff
@@ -88,7 +88,7 @@
         "prestart": "npm install",
         "start": "webpack --watch",
         "server": "webpack-dev-server -d",
-        "test": "mocha",
+        "test": "nyc --reporter=text mocha",
```

Comment:
We should maintain the ability to run the "fast test loop".

Reply:
Also, where is `nyc` added as a package? If it was installed globally, then we should make it local.

Reply:
1) Added `nyc` to dev-dependency in package.json 
2) for `to maintain fast test loop` since `--exit` option is not provided and `watch` is set to `true` inside mocha config, so to see the nyc report we would need to exit the mocha process. Sorry what's the recommendation?

Reply:
It's okay to have a separate script for each.

Reply:
Oh that's a good point. Thanks will do that

Reply:
Fixed